admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/ApiKeyAuthenticationProvider.kt

```diff
@@ -100,6 +113,8 @@ fun Authentication.Configuration.apiKey(
         } else if (principal == null) {
             challenge(AuthenticationFailedCause.InvalidCredentials)
         } else {
+            //TODO put each times?
```

Comment:
If cache is enabled, then yes, always when apiKey != null and principal != null

Reply:
as a workaround to disable caching we can set up cacheSize to 0.
Info was taken from docs:
"When size is zero, elements will be evicted immediately after being loaded into the cache. This can be useful in testing, or to disable caching temporarily without a code change. As eviction is scheduled on the configured executor, tests may instead prefer to configure the cache to execute tasks directly on the same thread."